api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -0,0 +1,230 @@
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static uk.gov.dhsc.htbhf.claimant.ClaimantServiceAssertionUtils.buildClaimRequestEntity;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimDTOTestDataFactory.aClaimDTOWithClaimant;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimantDTOTestDataFactory.aClaimantDTOWithExpectedDeliveryDate;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementMatchingChildrenAndPregnancy;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.TestConstants.NO_CHILDREN;
+
+/**
+ * Runs a claim through the entire lifecycle, preforming (limited) tests at each payment cycle to confirm the correct
+ * amounts have been paid and the correct emails sent.
+ */
+@SuppressWarnings("PMD.AvoidReassigningParameters")
+public class ClaimantLifecycleIntegrationTests extends ScheduledServiceIntegrationTest {
+
+    public static final int CYCLE_DURATION = 28;
+
+    @Autowired
+    TestRestTemplate restTemplate;
+
+    /**
+     * Run a simple claim through its entire lifecycle and confirm the correct events happen & right emails sent.
+     * Each vertical bar is the start of a payment cycle. 
+     *                                                     | one year
+     * |...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|
+     * | claim starts
+     * ..........| Due date & dob ...................................| child's first birthday
+     * ..................| New baby reported to DWP
+     * ....................| Backdated vouchers paid
+     * ........................................................| email about upcoming changes to payment
+     */
+    @Test
+    void shouldProcessClaimFromPregnancyToChildTurningFour() throws JsonProcessingException, NotificationClientException {
+        LocalDate expectedDeliveryDate = LocalDate.now().plusDays(70); // 2.5 cycles before due date
+
+        UUID claimId = applyForHealthyStartAsPregnantWomanWithNoChildren(expectedDeliveryDate);
+        assertFirstCyclePaidCorrectly(claimId);
+
+        // run through 4 cycles while pregnant (we pay vouchers for up to 12 weeks after due date
+        // - their due date is during the 3rd cycle so the claim would expire during 6th if no birth reported)
+        // TODO: HTBHD-2377 & HTBHF-1503 reminder email sent at second payment after due date
+        expectedDeliveryDate = progressThroughPaymentCyclesForPregnancy(expectedDeliveryDate, claimId, 4);
+
+        // child was born exactly on due date - one and a half cycles ago - notify in time to get backdated vouchers
+        List<LocalDate> childrenDob = fastForwardOneCycle(asList(expectedDeliveryDate));
```

Comment:
I thought `fastForwardOneCycle` dealt with children dobs and not a due date, that's confusing!